lib/src/DatePicker/DatePickerToolbar.tsx

```diff
@@ -26,50 +31,51 @@ export const useStyles = makeStyles(
 export const DatePickerToolbar: React.FC<ToolbarComponentProps> = ({
   date,
   views,
-  setOpenView,
   isLandscape,
-  openView,
+  title = 'SELECT DATE',
 }) => {
   const utils = useUtils();
-  const classes = useStyles();
+  const classes = useStyles({ isLandscape });
 
-  const isYearOnly = React.useMemo(() => isYearOnlyView(views as any), [views]);
-  const isYearAndMonth = React.useMemo(() => isYearAndMonthViews(views as any), [views]);
+  const dateTitle = React.useMemo(() => {
+    if (isYearOnlyView(views as DatePickerView[])) {
+      return utils.getYearText(date);
+    }
+
+    if (isYearAndMonthViews(views as DatePickerView[])) {
+      return utils.getMonthText(date);
+    }
+
+    return utils.getDatePickerHeaderText(date);
+  }, [date, utils, views]);
 
   return (
-    <PickerToolbar
-      isLandscape={isLandscape}
-      className={clsx({
-        [classes.toolbar]: !isYearOnly,
-        [classes.toolbarLandscape]: isLandscape,
-      })}
-    >
-      <ToolbarButton
-        variant={isYearOnly ? 'h3' : 'subtitle1'}
-        onClick={() => setOpenView('year')}
-        selected={openView === 'year'}
-        label={utils.getYearText(date)}
+    <PickerToolbar isLandscape={isLandscape} className={classes.toolbar}>
+      <Typography
+        color="inherit"
+        variant="overline"
+        children={title}
```

Comment:
A funny coding style, was it on purpose 🙃?

Reply:
It allows get rid of unnecessary spaces when passing down children’s

Reply:
So it's to optimize for readability when developing?

Reply:
Both. But actually this 
```jsx
<div> {children} </div>
```

Will render 2 spaces before and after the children. And also looks more like spreading things down instead of rendering.